### ISSUE TITLE
"View all entries from the same date" displayed perplexingly on Create Entry success landing page

### ISSUE THREAD

**Issue by kaberett** [dreamwidth/dw-free#1665]
(Semi-tested using Beta update page.)

When updating with "Use the current time" and "Don't show on Reading pages" checked, the second item in the "Now you can..." list was "View all entries from the same date".

When updating with "Use the current time" but _not_ "Don't show on Reading pages" checked, "View all entries from the same date" does not display.

When editing an entry with "Don't show on the Reading page" but _not_ "Use the current time" checked, "View all entries from the same date" does not display.

It's not clear to me why this might be intentional, or what is gained from this inconsistency due to these specific options.

**Comment by rahaeli** [comment]
Looking at the code, it appears that the "view all entries from the same date" link is the string .new.links.backdated or .success.links.backdated and does indeed only show if you have the don't-show-on-reading-pages-formerly-known-as-backdating checkbox ticked. Looks like this is a confusion stemming from the fact that "don't show on reading pages" used to need to be checked in order to post an entry in the past (backdated); we changed it so that "backdate" is no longer a necessity (since we removed the 'you're trying to post in the past' check, we renamed that option from 'backdate' to 'don't show on reading pages') and that kind of abstracted away the reasoning behind why "view all entries from the same date" only showed if something was backdated.

The logic around that is in lines 1011-1019 of cgi-bin/DW/Controller/Entry.pm and can probably just be taken out of the if-backdated block.
